packages/react-dev-utils/webpackHotDevClient.js

```diff
@@ -25,6 +25,16 @@ var launchEditorEndpoint = require('./launchEditorEndpoint');
 var formatWebpackMessages = require('./formatWebpackMessages');
 var ErrorOverlay = require('react-error-overlay');
 
+ErrorOverlay.listenToOpenInEditor(function OpenInEditor(errorLocation) {
```

Comment:
IMO, `listenToOpenInEditor` is a little ambiguous. e.g. can you listen with more than one function, or is this action reversible?
Questions people might have in mind if they're thinking of an EventEmitter (listeners/observers).

What about `setEditorCallback` or `setEditorHandler`?

Reply:
I see your point.  Just I couldn't think of any better name than this. Let's go with `setEditorHandler`.